recipes/triton/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "1.1.2" %}
+
+package:
+  name: triton
+  version: {{ version }}
+
+source:
+  url: https://github.com/openai/triton/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 44040696460f1d02a7251dd59468d7405982b55e112a9999ec759718066f7708
+  patches:
+    - patches/0001-patch-include-paths-in-setup.py.patch
+    - patches/0002-add-missing-header.patch
+    - patches/0003-properly-point-to-triton-includes.patch
+    - patches/0004-point-to-system-CUDA-instead-of-vendored-header.patch
+
+build:
+  number: 0
+  # TODO: windows support should be available from next version;
+  #       CPU-only support still under development
+  skip: true  # [win or cuda_compiler_version == "None"]
```

Comment:
I don't understand. Why do we have a cbc yaml file including OS X targets, while this recipe requires CUDA? macOS has no CUDA support these days.

Reply:
The CBC was an artefact from trying to get the CPU builds to work until I found out that upstream does not support that at all at the moment - will remove.